tools/topology/sof-icl-rt711-rt1308-rt715-hdmi.m4

```diff
@@ -53,7 +53,8 @@ PIPELINE_PCM_ADD(sof/pipe-volume-capture.m4,
 
 # Low Latency playback pipeline 3 on PCM 2 using max 2 channels of s32le.
 # Schedule 48 frames per 1000us deadline on core 0 with priority 0
-PIPELINE_PCM_ADD(sof/pipe-volume-demux-playback.m4,
+PIPELINE_PCM_ADD(ifdef(`MONO',`sof/pipe-volume-playback.m4',
+`sof/pipe-volume-demux-playback.m4',)
```

Comment:
there's already an ifdef on line 61, it'd probably be a good thing to have clearer separation between the cases.
Also we may want to conditionally disable aggregation, as we do e.g. for the kernel.

Reply:
@bardliao what I meant was

````
if MONO
   pipelines, etc
else
  if aggregation
      pipelines, etc
  else 
      pipelines, pcm, etc.
  endif
endif
````

Reply:
@plbossart Done. But a little bit different to your example.